third_party/move/move-compiler-v2/tests/ability-checker/ability_violation.exp

```diff
@@ -226,6 +226,16 @@ fun ability::no_key($t0: address) {
   5: return ()
 }
 
+
+Diagnostics:
+error: cannot immutable borrow global `ability::Impotent` since other mutable references exist
```

Comment:
I note that `$t3` is not live after the first borrow, so why is there a conflict with the second borrow?

Btw, you might want to add a fun to your example code with that stores references in variables, so we can see the difference in behavior.

Reply:
Done. The cause is that the second iteration of liveness analysis considers the local in a drop as used.

Reply:
You say "Done" but I don't see any changes to `.move` files in this PR.

Reply:
By done, I mean the above wrong diagnostics have been resolved. I didn't add a fun such that the error should happen because it's likely already included in the tests of the borrow checker.

Reply:
Please add a fun to your example code with that stores references in variables, so we can see the difference in behavior.